Bottom: 5dcb6450cc8c676d74e87ba8c35f589fa1b828f6
Top:    52f6c05d9c82e493fab96a109790e634d11f3948
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 00:31:27 +0100

Refresh of w-c-split-gathering-routines-f-0

---

diff --git a/src/draw.c b/src/draw.c
index ea5e9ce..fe7f7f2 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -755,25 +755,6 @@ element_callback (const BoxType * b, void *cl)
   return 1;
 }
 
-static void
-draw_element_name (ElementType *element)
-{
-  if (gui->gui && TEST_FLAG (HIDENAMESFLAG, PCB))
-    return;
-  if (TEST_FLAG (HIDENAMEFLAG, element))
-    return;
-  if (doing_pinout || doing_assy)
-    gui->set_color (Output.fgGC, PCB->ElementColor);
-  else if (TEST_FLAG (SELECTEDFLAG, &ELEMENT_TEXT (PCB, element)))
-    gui->set_color (Output.fgGC, PCB->ElementSelectedColor);
-  else if (FRONT (element))
-    gui->set_color (Output.fgGC, PCB->ElementColor);
-  else
-    gui->set_color (Output.fgGC, PCB->InvisibleObjectsColor);
-  DrawTextLowLevel (&ELEMENT_TEXT (PCB, element), PCB->minSlk);
-}
-
-
 /* ---------------------------------------------------------------------------
  * draws one non-copper layer
  */
